src/public-galaxy-servers/pitagora-galaxy/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Pitagora-Galaxy"
+url: "http://try.pitagora-galaxy.org/galaxy/"
+server_group: "general"
+server_links: 
+  - link_url: "http://try.pitagora-galaxy.org/galaxy/"
+    link_text: "Pitagora-Galaxy Server"
+  - link_url: "http://www.pitagora-galaxy.org/about_en"
+    link_text: "Pitagora-Galaxy Project"
+  - link_url: "/src/va/pitagora-galaxy/index.md"
```

Comment:
This link doesn't look like it's working when rendered, to me, but I haven't had a chance to dig yet.  I will.

Reply:
My apologies, that is not true. ⬆️ 

There is a duplication:
http://127.0.0.1:8080/public-galaxy-servers/pitagora-galaxy/
http://127.0.0.1:8080/va/pitagora-galaxy/

But the `src` prefixed link won't work. I think we want to remove the dup anyways, so I will do so.

Reply:
Yeah, it was fixed in subsequent commits.  All looks good to me now.

Reply:
It is link at the individual server page (http://127.0.0.1:8080/public-galaxy-servers/pitagora-galaxy/) and it goes to http://127.0.0.1:8080/src/va/pitagora-galaxy/index.md which is 404. Whit applies to more pages I think, as per @tnabtaf  post merge comment.